menas/ui/components/schema/schemaDetail.controller.js

```diff
@@ -122,6 +122,8 @@ sap.ui.define([
         this._eventBus.publish("schemas", "list");
         // nav back to info
         this.byId("schemaIconTabBar").setSelectedKey("info");
+      } else if (status === 401) {
```

Comment:
I should have mentioned in the bug report that I get 403, not 401 when this happens.

Reply:
My bad, I was too lazy to wait for a timeout :)